core/src/sigverify_stage.rs

```diff
@@ -445,7 +445,7 @@ mod tests {
         for _ in 0..batches.len() {
             if let Some(batch) = batches.pop() {
                 sent_len += batch.packets.len();
-                packet_s.send(batch).unwrap();
+                packet_s.send(vec![batch]).unwrap();
```

Comment:
hmm why is this change necessary to wrap this in a vector now? It seems like sigverify is just unpacking this on the other end: https://github.com/solana-labs/solana/blob/7aba3ad2afbd92a2321d9eac9d670a72faba42ca/core/src/sigverify_stage.rs#L235